packages/legend-studio/src/stores/editor-state/element-editor-state/mapping/PureInstanceSetImplementationState.ts

```diff
@@ -126,9 +127,96 @@ export class PurePropertyMappingState extends PropertyMappingState {
   }
 }
 
+export class PureInstanceSetImplementationFilterState extends LambdaEditorState {
+  filter: RawLambda | undefined;
+  editorStore: EditorStore;
+  instanceSetImplementationState: PureInstanceSetImplementationState;
+  constructor(
+    instanceSetImplementationState: PureInstanceSetImplementationState,
+    editorStore: EditorStore,
+    filter?: RawLambda,
+  ) {
+    super('true', LAMBDA_PIPE);
+
+    makeObservable(this, {
+      filter: observable,
+      editorStore: observable,
+    });
+
+    this.editorStore = editorStore;
+    this.filter = filter;
+    this.instanceSetImplementationState = instanceSetImplementationState;
+  }
+
+  get lambdaId(): string {
```

Comment:
Is this the correct way to generate the lambdaId?
should it be based on some actual path in the graph? @akphi @MauricioUyaguari